phoenix-tools/src/it/java/org/apache/phoenix/schema/SchemaExtractionToolIT.java

```diff
@@ -58,156 +130,251 @@ public void testCreateTableStatement_tenant() throws Exception {
+        queries.add(query);
+        String result = runSchemaExtractionTool(schemaName, tableName, null, queries);
+        Assert.assertEquals(query.toUpperCase(), result.toUpperCase());
+    }
 
-        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+    @Test
+    public void testCreateTableWithNonDefaultColumnFamily() throws Exception {
+        String tableName = generateUniqueName();
+        String schemaName = generateUniqueName();
+        String pTableFullName = SchemaUtil.getQualifiedTableName(schemaName, tableName);
+        String query = "create table " + pTableFullName +
+                "(a_char CHAR(15) NOT NULL, " +
+                "b_char CHAR(10) NOT NULL, " +
+                "\"av\".\"_\" CHAR(1), " +
+                "\"bv\".\"_\" CHAR(1), " +
+                "\"cv\".\"_\" CHAR(1), " +
+                "\"dv\".\"_\" CHAR(1) CONSTRAINT PK PRIMARY KEY (a_char, b_char)) " +
+                "TTL=1209600, IMMUTABLE_ROWS=true, IMMUTABLE_STORAGE_SCHEME=ONE_CELL_PER_COLUMN, SALT_BUCKETS=16, MULTI_TENANT=true";
+        List<String> queries = new ArrayList<String>(){};
+        queries.add(query);
+        String result = runSchemaExtractionTool(schemaName, tableName, null, queries);
+        Assert.assertEquals(query.toUpperCase(), result.toUpperCase());
+    }
 
-            String pTableFullName = SchemaUtil.getQualifiedTableName(schemaName, tableName);
-            conn.createStatement().execute("CREATE TABLE "+pTableFullName + "(k BIGINT NOT NULL PRIMARY KEY, "
-                    + "v1 VARCHAR, v2 VARCHAR)"
-                    + properties);
-            String viewFullName = SchemaUtil.getQualifiedTableName(schemaName, viewName);
-            String viewFullName1 = SchemaUtil.getQualifiedTableName(schemaName, viewName+"1");
+    @Test
+    public void testCreateTableWithUniversalCFProperties() throws Exception {
+        String tableName = generateUniqueName();
+        String schemaName = generateUniqueName();
+        String properties = "KEEP_DELETED_CELLS=TRUE, TTL=1209600, IMMUTABLE_STORAGE_SCHEME=ONE_CELL_PER_COLUMN, REPLICATION_SCOPE=1";
+        String pTableFullName = SchemaUtil.getQualifiedTableName(schemaName, tableName);
+        String query = "create table " + pTableFullName +
+                "(a_char CHAR(15) NOT NULL, " +
+                "b_char CHAR(10) NOT NULL, " +
+                "\"av\".\"_\" CHAR(1), " +
+                "\"bv\".\"_\" CHAR(1), " +
+                "\"cv\".\"_\" CHAR(1), " +
+                "\"dv\".\"_\" CHAR(1) CONSTRAINT PK PRIMARY KEY (a_char, b_char)) " + properties;
+        List<String> queries = new ArrayList<String>(){};
+        queries.add(query);
+        String result = runSchemaExtractionTool(schemaName, tableName, null, queries);
+        Assert.assertEquals(query.toUpperCase(), result.toUpperCase());
+    }
 
+    @Test
+    public void testCreateTableWithDefaultCFProperties() throws Exception {
```

Comment:
even this can be clubbed with above one as there is only one extra property here.